gridstatus/miso.py

```diff
@@ -22,8 +24,10 @@ class MISO(ISOBase):
 
     name = "Midcontinent ISO"
     iso_id = "miso"
+    # TODO: these next two lines seem to be contradictory
     # miso spans multiple timezones, so picking central
     # all parsing is done in EST since that is what api returns
+    parsing_timezone = "EST"
```

Comment:
Is there a source for this? (I didn't change the logic, just assigned it to a variable)

Reply:
for example, if you open this [file]([20240331_da_expost_lmp.csv](https://docs.misoenergy.org/marketreports/20240331_da_expost_lmp.csv)) you can see a note that says "All Hours-Ending are Eastern Standard Time (EST)". or, if you look that the [api we use for fuel mix](https://api.misoenergy.org/MISORTWDDataBroker/DataBrokerServices.asmx?messageType=getfuelmix&returnType=json) it's in EST (even though we are currently in EDT)

also, found these two sources why googling

* https://www.rtoinsider.com/25291-ferc-oks-miso-use-of-eastern-standard-time-in-day-ahead-market/
* https://github.com/moment/luxon/discussions/1270

Reply:
Thanks, I'll add these in a comment. 

I wonder if this comment, "# miso spans multiple timezones, so picking central" is just outdated?

Reply:
ya, it is. 

I think for default timezone. we probably want it to be EST too. that goal of the variable is really to be the "market timezone".